app/models/compute_resources/foreman/model/ec2.rb

```diff
@@ -52,9 +52,18 @@ def find_vm_by_uuid(uuid)
 
     def create_vm(args = { })
       args = vm_instance_defaults.merge(args.to_h.symbolize_keys).deep_symbolize_keys
+      # Merge AWS EC2 tags
+      new_tags = {}
       if (name = args[:name])
-        args[:tags] = {:Name => name}
+        new_tags = {:Name => name}
       end
+      unless args[:tags].nil?
+        args[:tags].each_line do |line|
```

Comment:
If you change this line to:
```suggestion
        args[:tags]&.each_line do |line|
```
you can drop the condition and make the linter happy :)

Reply:
Right, it seems to keep working, rebasing...